Portal/vip-portal/src/main/resources/vm/wrapper.vm

```diff
@@ -19,6 +19,18 @@ function error {
   echo [ ERROR - $D ] $* >&2
 }
 
+function checkBosh {
+  export PATH=$PATH:$HOME/.local/bin
+  BOSH=`which bosh`
```

Comment:
You're right, I didn't think of other tools that may have the same name. I thought more of the environments where installing boutiques may be troublesome (see the issues I had on local VMs with CentOS6 and Python 2.6). It would therefore be useful to have boutiques pre-installed. Do you see a way of checking that the existing "bosh" is the correct one?

Reply:
btw, I've just learned that "command' is the way to do this, not "which". See https://stackoverflow.com/questions/592620/how-to-check-if-a-program-exists-from-a-bash-script 
I don't know how to check that "bosh" is indeed our "bosh" if the command exists, except to try and run it for a simple command. `bosh create foo.sh` sounds like the easiest and cheapest one at the moment.

Reply:
Cool, thanks for the "command" tip!